### PR TITLE
Remove no cover lines from under_cached_property.__get__

### PR DESCRIPTION
**Pull request by bdraco**
These should already be handled by covdefaults

### CODE REVIEW

File: src/propcache/_helpers_py.py [hunk 1]
```diff
@@ -46,14 +46,10 @@ def __init__(self, wrapped: Callable[..., _T]) -> None:
...
-    def __get__(  # pragma: no cover
-        self, inst: None, owner: Optional[Type[Any]] = None
-    ) -> Self: ...  # pragma: no cover
+    def __get__(self, inst: None, owner: Optional[Type[Any]] = None) -> Self: ...
 
     @overload
-    def __get__(  # pragma: no cover
-        self, inst: _TSelf[_T], owner: Optional[Type[Any]] = None
-    ) -> _T: ...  # pragma: no cover
+    def __get__(self, inst: _TSelf[_T], owner: Optional[Type[Any]] = None) -> _T: ...
```
Comment (webknjaz) [inline src/propcache/_helpers_py.py:52]:
```suggestion
    def __get__(self, inst: _TSelf[_T], owner: Optional[Type[Any]] = None) -> _T:
        ...
```

File: src/propcache/_helpers_py.py [hunk 2]
```diff
@@ -46,14 +46,10 @@ def __init__(self, wrapped: Callable[..., _T]) -> None:
         self.name = wrapped.__name__
 
     @overload
-    def __get__(  # pragma: no cover
-        self, inst: None, owner: Optional[Type[Any]] = None
-    ) -> Self: ...  # pragma: no cover
+    def __get__(self, inst: None, owner: Optional[Type[Any]] = None) -> Self: ...
```
Comment (webknjaz) [inline src/propcache/_helpers_py.py:49]:
```suggestion
    def __get__(self, inst: None, owner: Optional[Type[Any]] = None) -> Self:
        ...
```

**Review by webknjaz — COMMENTED** [review]
I think Covdefaults only handles ellipsis on a separate line. Otherwise, it'd exclude too much.

File: src/propcache/_helpers_py.py [hunk 3]
```diff
@@ -46,14 +46,10 @@
...
```
[the remark this answers is not in the document]
Comment (webknjaz) [reply src/propcache/_helpers_py.py:49]: Required for typing

**Comment by bdraco** [comment]
> I think Covdefaults only handles ellipsis on a separate line. Otherwise, it'd exclude too much.

I tried that but pre-commit moved them back

**Comment by webknjaz** [comment]
It looks like this should also cover the same line: https://pypi.org/project/covdefaults/

**Comment by webknjaz** [comment]
@bdraco that's not a pytest+coveragepy partial, though: https://app.codecov.io/gh/aio-libs/propcache/pull/48/blob/src/propcache/_helpers_py.py?flags%5B0%5D=pytest#L54. You can see it with only the `pytest` flag set. It's coming from MyPy: https://app.codecov.io/gh/aio-libs/propcache/pull/48/blob/src/propcache/_helpers_py.py?flags%5B0%5D=MyPy.

To see MyPy coverage locally, do `python3 -Im webbrowser $(ls -1 .tox/.tmp/.mypy/python-3.*/index.html | head -n 1)` after running pre-commit.

**Comment by bdraco** [comment]
I get it, but the combined view is a bit misleading because it looks like there is a problem but there isn't.   I'm not sure there is anything to do about it though.

**Comment by webknjaz** [comment]
yeah, this probably needs research

**Pull request merged**